src/ostorlab/cli/scan/list/list.py

```diff
@@ -3,12 +3,14 @@
 Example of usage:
     - ostorlab scan list --source=source."""
 
+from typing import Union
```

Comment:
It has been recommended to just use the '|'

Reply:
in oxo we have a workflow for python 3.9 which doesnt support the `|` notation.